### PR TITLE
Yuval/adding req nvme

### PR DESCRIPTION
**Pull request by yuval-lb**
3 things
1. nvme drivers to load nvme devices
2. pci utils for easy parsing
3. requests python package to communicate with outside.

*NOTE - I still have a problem with nvme currently, I need to login the machine and run mdev -s.
the execution of mdev -s in the init script is not working.

if will not be able to overcome this - when we will inspect the nvme devices we will run "mdev -s"

### CODE REVIEW

File: init [hunk 1]
```diff
@@ -15,8 +15,9 @@ echo ifconfig -a:
 /usr/sbin/busybox modprobe ahci
```
Comment (yuval-lb) [inline init]: this seems to work

File: init [hunk 2]
```diff
@@ -14,7 +14,9 @@ echo ifconfig -a:
 /usr/sbin/busybox modprobe sd_mod
 /usr/sbin/busybox modprobe ahci
 /usr/sbin/busybox modprobe ext4
+/usr/sbin/busybox modprobe nvme
 echo lsmod:
 /usr/sbin/busybox lsmod
 export PYUDEV_UDEV_LIBRARY_NAME=libudev.so.1
 exec /bin/python2.7 -m inaugurator.main
+/usr/sbin/busybox mdev -s
```
Comment (adir-lb) [inline init]: @yuval-lb what is exactly the issue with running this command in the init script?

File: Makefile.build [hunk 3]
```diff
@@ -93,6 +93,8 @@ build/inaugurator.thin.initrd.dir: ${PYTHON_DIST_LOCAL_PATH} build/osmosis-1.0.l
 	echo "Copying pika:"
 	DEST=$@.tmp sh/relative_copy_glob.sh /usr/lib/python2.7/site-packages/pika/*.py
 	DEST=$@.tmp sh/relative_copy_glob.sh /usr/lib/python2.7/site-packages/pika/adapters/*.py
+	rsync -avzh /usr/lib64/python2.7/ build/inaugurator.thin.initrd.dir.tmp/usr/lib64/python2.7/
+	rsync -avzh /usr/lib/python2.7/ build/inaugurator.thin.initrd.dir.tmp/usr/lib/python2.7/
```
Comment (adir-lb) [inline Makefile.build]: why are copying python2.7 directory from the host/docker image to inaugurator build dir?

**Comment by sashas-lb** [comment]
@Yuval Brave <yuval@lightbitslabs.com>  can you please post the difference
in image sizes .. before and after ?

**Comment by yuval-lb** [comment]
@sashas-lb currently its 19M after the change 30M

File: Makefile.build (hunk 3 above)
Comment (yuval-lb) [reply Makefile.build]: requests have many dep's copying requests and all its sub dir is not enough. after 6-7 times of each time, adding different package, I decided to copy all.

File: init (hunk 2 above)
Comment (yuval-lb) [reply init]: @adir-lb the fixup is working.

but the problem here was, the nvme was discovered, but /dev/ did not show any nvme device.

i assume mdev -s ran before modprobe nvme finished.
adding "wait" did the job

**Pull request merged**